MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -90,7 +90,7 @@ open class BaseInstructionsBannerView: UIControl {
         }
     }
     
-    func set(_ instruction: VisualInstructionBanner?) {
+    public func set(_ instruction: VisualInstructionBanner?) {
```

Comment:
Let's rename this too to make it more inline with `updateDistance(for:)` and make it more clear what you're updating.